bindings/python/README.rst

```diff
@@ -42,11 +42,23 @@ suited for in-memory analytical processing applications.
 
 Installing PyMongoArrow
 =======================
-PyMongoArrow is available on PyPI:::
+PyMongoArrow is available on PyPI::
 
   $ python -m pip install pymongoarrow
 
-PyMongoArrow wheels are available for macOS and Linux on x86_64 architectures.
+To use PyMongoArrow with MongoDB Atlas' ``mongodb+srv://`` URIs, you will
+need to also install PyMongo with the ``srv`` extra::
+
+  $ python -m pip install pymongo[srv]<4 pymongoarrow
```

Comment:
Suggest adding quotes to make this copy/pastable in all terminals: `'pymongo[srv]<4'`

Reply:
Done.